templates/zerver/help/user-groups.md

```diff
@@ -27,6 +27,11 @@ trying to send a message to a group of people, you'll want to either
 1. Add or remove users (including yourself). Click outside the box
    to save.  Zulip will notify everyone who is added or removed.
 
+!!! warn ""
+    **Note**: If you remove yourself from a user group,
+    you may not have permissions to continue modifying
```

Comment:
How about

"you may not have permissions to continue modifying" -> "you may no longer have permissions to modify" ?